### ISSUE TITLE
Hitting download with manager doesn't open Nexus mod manager anymore

### ISSUE THREAD

**Issue by User104776537-205** [Nexus-Mods/Nexus-Mod-Manager#913]
A while ago (a few updates ago) you used to be able to automatically open NMM and download mods by hitting the DL with manager button on most mods on the nexus. now i literally have to reinstall NMM every single time i want to install new mods with the program because it just doesn't work 
(not a pun). thats the only way i can get it to work properly. not a terrible experience, but i would very much like that whole auto open NMM when you want to download a mod. like it was in the good ol days before that Vortex crap appeared. any chance this could be fixed? because i think the nexus guys are screwing over the NMM tool so that people will use there stupid program instead. idk just a thought.

**Comment by jbostrus** [comment]
Do you have Vortex and/or ModOrganizer2 installed? Both of those, and probably any other mod manager, will try to take over the Nexus links. Vortex has an option to take it over, but then also has a toggle to put it back if you choose. In any case this is not a Nexus Mod Manager bug.

You can have NMM reassociate it's link grabbing by running it as Administrator, opening the options screen using the gear icon, goto the OS Settings tab, and check the Associate with NXM URLs. But if you use any other tools that also try to do that it will get lost again.

**Comment by User104776537-205** [comment]
> Do you have Vortex and/or ModOrganizer2 installed? Both of those, and probably any other mod manager, will try to take over the Nexus links. Vortex has an option to take it over, but then also has a toggle to put it back if you choose. In any case this is not a Nexus Mod Manager bug.
> 
> You can have NMM reassociate it's link grabbing by running it as Administrator, opening the options screen using the gear icon, goto the OS Settings tab, and check the Associate with NXM URLs. But if you use any other tools that also try to do that it will get lost again.

No, i don't use Vortex, they just made it look...idk more complicated looking, don't like it. so i installed the community version for its simplicity and LOOT for sorting mods. when i reinstall NMM it works fine i can download all the mods i want. but when i update it, it seems to break it, and it no longer receives a signal from the nexus site or whatever when i click dl with manager. so i have to reinstall it back to the "older version". idk just don't update it anymore i guess.
